lesson5/models/user.js

```diff
@@ -0,0 +1,41 @@
+const mongoose = require('mongoose')
+const Schema = mongoose.Schema
+
+mongoose.connect('mongodb://localhost/todos')
+
+const userSchema = new Schema({
+  email: {
+    type: String,
+  },
+  password: {
+    type: String,
+  },
+  facebookId: {
+    type: String,
+  },
+})
+
+userSchema.statics.findOrCreate = async function(service, userData) {
```

Comment:
It seems like we're having already this method

Reply:
It's seems than I have older version of mongo, that doesn't have new methods. Should update it>

Reply:
Do you mean mongoose?

Reply:
You're right, it's mongoose, but needs additional plugin as i found out https://github.com/drudge/mongoose-findorcreate
Without it node doesn't see the function, says function not found.

Reply:
Actually, I mean [this](http://mongoosejs.com/docs/api.html#findoneandupdate_findOneAndUpdate). You need to specify the "upsert" parameter as well

Reply:
Thanks, I will discover it.